roles/edpm_sshd/tasks/configure.yml

```diff
@@ -74,11 +74,16 @@
          {% if edpm_sshd_banner_enabled %}
          {% set _ = edpm_sshd_server_options.__setitem__('Banner', '/etc/issue') %}
          {% endif %}
-         {% if edpm_sshd_motd_enabled %}
-         {% set _ = edpm_sshd_server_options.__setitem__('PrintMotd', 'yes') %}
-         {% endif %}
          {{ edpm_sshd_server_options }}
 
+    - name: Set sshd motd when enabled
+      ansible.builtin.lineinfile:
+        path: /etc/pam.d/sshd
+        regexp: "^session.*optional.*pam_motd.so"
+        line: "session    optional     pam_motd.so motd=/etc/motd"
```

Comment:
LGTM, i already have the line in this file in my RHEL 9.2 edpm node, but without the `motd=/etc/motd`, that is the default, so that part is likely not needed. But, I'm fine either way.

Reply:
Yeah, so the issue this solves is that `PrintMotd` defaults to `yes` in `sshd_config`. So if we don't explicitly set it to `no` then the motd is duplicated.

Explicitly telling pam the exact file to use results in the single message being printed instead of the duplication. But, I did some more digging to make sure I was answering the questions on the BZ accurately, and I actually think that this might be a better solution to the problem:
https://github.com/openstack-k8s-operators/edpm-ansible/pull/838